rpcs3/emucore.vcxproj

```diff
@@ -690,4 +691,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
Unnecessary leenux user changes.
Will be reverted by the next normal edit anyway

Reply:
These kind of things are automatically done by the code editor, will try to revert

Reply:
I'm not being able to remove this, already tried saving without trailing whitespace and made sure the file is on CRLF